chains/solana/contracts/programs/ccip-common/src/v1.rs

```diff
@@ -2,20 +2,32 @@ use anchor_lang::prelude::*;
 use solana_program::address_lookup_table::state::AddressLookupTable;
 
 use crate::{
-    context::TokenAccountsValidationContext, router_accounts::TokenAdminRegistry, CommonCcipError,
+    context::TokenAccountsValidationContext, router_accounts::TokenAdminRegistry, seed,
+    CommonCcipError,
 };
 
+pub const MIN_TOKEN_POOL_ACCOUNTS: usize = 13; // see TokenAccounts struct for all required accounts
+
 pub struct TokenAccounts<'a> {
     pub user_token_account: &'a AccountInfo<'a>,
     pub token_billing_config: &'a AccountInfo<'a>,
     pub pool_chain_config: &'a AccountInfo<'a>,
+    pub lookup_table: &'a AccountInfo<'a>,
+    pub token_admin_registry: &'a AccountInfo<'a>,
```

Comment:
those two weren't part of the token accounts? 🤔

Reply:
It was, but this method was "swallowing" that value and not returning it. So, for this method to be clearer, I made it return all the accounts it gets, and thus the `MIN_TOKEN_POOL_ACCOUNTS` constant is also more consistent (as that one relates to the input accounts, and the actual struct was only listing the accounts this method returns)